mod_reforged/hooks/entity/tactical/enemies/zombie_knight_bodyguard.nut

```diff
@@ -2,8 +2,8 @@
 	q.onInit = @() function()
 	{
 	    this.zombie_knight.onInit();
-		// local b = this.m.BaseProperties;
-		// b.IsSpecializedInShields = true;
+		local b = this.m.BaseProperties;
+		b.IsSpecializedInShields = false;	// this is the only enemy where this property in vanilla is hard-coded to true
```

Comment:
We are already overwriting the vanilla function and the default value for this variable in character properties _is_ `false`. So I don't see it necessary that we should manually overwrite this. We completely overwrite the vanilla function, and show these lines as commented out to show that this is the part of the vanilla function that we "removed".